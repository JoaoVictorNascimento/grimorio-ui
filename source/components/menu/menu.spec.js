import '../../../internals/test/helper';

import Menu from './index';

/** @test {Menu} */
describe('Menu component', () => {
/** @test {Menu#render} */
  describe('#render', () => {
    it('render correctly', () => {
      const wrapper = shallow(
        <Menu />
      );
      expect(wrapper.length).to.equal(1);
    });
  });
});
