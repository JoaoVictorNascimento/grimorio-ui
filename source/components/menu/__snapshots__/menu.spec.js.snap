// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu component #render render with children 1`] = `"<ul class=\\"menu default\\"><div>render com filho</div></ul>"`;

exports[`Menu component #render render with items 1`] = `"<ul class=\\"menu teste accordionMenu dark\\"><li title=\\"\\" class=\\"menuItem\\"><i style=\\"vertical-align:middle;font-size:16px\\" class=\\"material-icons icon iconLeft\\" align=\\"middle\\">dashboard</i>Dashboard<ul class=\\"menu default\\"><li title=\\"\\" class=\\"menuItem\\">Default</li><li title=\\"\\" class=\\"menuItem\\">eCommerce</li><li title=\\"\\" class=\\"menuItem\\">News Portal</li></ul></li><li title=\\"\\" class=\\"menuItem\\"><i style=\\"vertical-align:middle;font-size:16px\\" class=\\"material-icons icon iconLeft\\" align=\\"middle\\">insert_chart</i>Charts<ul class=\\"menu default\\"><li title=\\"\\" class=\\"menuItem\\">Test</li></ul></li></ul>"`;
